### PR TITLE
Goal view - Added first interaction.

### PR DESCRIPTION
**Pull request by amitbharadwaj**

### CODE REVIEW

File: src/org/missionassetfund/apps/android/activities/MainActivity.java [hunk 1]
```diff
@@ -10,46 +11,51 @@
...
+          }
+        } else {
+          if (fragment != null) {
+            transaction.hide(fragment);
+          }
         }
-        transaction.commit();
+      }
+    } catch (Exception e) {
+      e.printStackTrace();
```
Comment (fcduarte) [inline src/org/missionassetfund/apps/android/activities/MainActivity.java]: We should `Log.d(tag, message, exception)` here

**Comment by fcduarte** [comment]
Reviewed. We're having problems when one of us is formatting code .. Could someone grab a eclipse code standard XML and share with the others?

**Comment by fcduarte** [comment]
hey @amitbharadwaj  me again .. So I checked out the code and I have a few more comments. It was cool the way you organized the fragments but for having the expected behaviour of the mockups you should create an activity for lending circles. That way we'll have the up button on the action bar, plus button for add a new goal and a animated transition between the screens...

What do you think? Any thoughts @josemontesdeoca ?

**Comment by josemontesdeoca** [comment]
Agreed. From the mock interaction this should be an Activity, once the user click the box it should start an intent. This will make things a bit easier at this point.

**Comment by amitbharadwaj** [comment]
Yes. We can play with activity vs fragment later.

**Pull request merged**